docs/index.md

```diff
@@ -38,41 +38,39 @@ resource "ec_deployment" "my_deployment" {
 
 ## Authentication
 
-The Elastic Cloud provider offers two methods of authentication against the remote API. Depending on the target environment, one or all can be used. By default the `endpoint` which the provider will target is the Public API of the Elasticsearch Service (ESS) offering.
+The Elastic Cloud (EC) provider offers two methods of authentication against the remote API: `apikey` or a combination of `username` and `password`. Depending on the environment, one or both can be used. The Public API of Elasticsearch Service (ESS) is the default `endpoint` that the provider will target.
```

Comment:
```suggestion
The Elastic Cloud Terraform provider offers two methods of authentication against the remote API: `apikey` or a combination of `username` and `password`. Depending on the environment, one or both can be used. The Public API of Elasticsearch Service (ESS) is the default `endpoint` that the provider will target.
```

I've never seen `EC` as a valid short form for Elastic Cloud and our style guide says it's _verboten_:

![image](https://user-images.githubusercontent.com/15148011/98179685-2a721c00-1eb4-11eb-82b9-83bd1ff11891.png)